### DISCUSSION TITLE
Is that possible for chai to support float32?

### DISCUSSION THREAD

**Discussion by JinyuanSun** [chaidiscovery/chai-lab#95] [Q&A] [answered]

**Comment by arogozhnikov** [comment, marked as answer]
Hi Jinyuan,

technically it is possible to support float32, but doesn't make any practical sense - you'll not get good speed and will be bottlenecked by memory anyway

**Reply by JinyuanSun** [reply]
I agree, but people might have some old gpus like v100.
